### DISCUSSION TITLE
Always-on-display

### DISCUSSION THREAD

**Discussion by julianfairfax** [InfiniTimeOrg/InfiniTime#1923] [Ideas]
An always-on-display would be nice. As the PineTime has an OLED screen, this should be possible?

**Comment by FintasticMan** [comment]
The PineTime doesn't have an OLED screen, it's an LCD display. There is some work being done towards an always-on display though: #1869.

**Reply by julianfairfax** [reply]
My mistake, I was led to believe that by the Wikipedia page.

**Comment by mark9064** [comment]
Battery life ~48h is the best possible (including ~8h screen off time per day for sleep)

**Comment by julianfairfax** [comment]
See #1869
